scrapers/ESP/index.js

```diff
@@ -1,38 +1,128 @@
+import { LocalDate, DateTimeFormatter } from '@js-joda/core';
 import * as fetch from '../../lib/fetch.js';
-import * as parse from '../../lib/parse.js';
 import * as transform from '../../lib/transform.js';
-import * as rules from '../../lib/rules.js';
-
-// Set county to this if you only have state data, but this isn't the entire state
-// const UNASSIGNED = '(unassigned)';
 
 const scraper = {
   country: 'ESP',
-  url: 'https://opendata.arcgis.com/datasets/48fac2d7de0f43f9af938852e3748845_0.csv',
+  url: 'https://github.com/datadista/datasets/tree/master/COVID%2019',
   priority: 1,
+  headless: false,
+  timeseries: true,
   aggregate: 'state',
```

Comment:
Can you cite whoever is curating this with the `curators` field? I understand it's community sources, is there a web page for the community? Do they cite their sources?

Reply:
I've added the `curators` field. The sources are Spain's Ministry of Health and Department of National Security, which publishes in PDF form. Is there a good place to note that in the scraper's metadata?